src/main/java/org/symphonyoss/symphony/entityjson/EntityJsonParserBuilder.java

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ *
+ * Copyright 2017 Symphony Communication Services, LLC.
+ *
+ * Licensed to The Symphony Software Foundation (SSF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The SSF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.symphonyoss.symphony.entityjson;
+
+/**
+ * Builder for EntityJsonParser.
+ * 
+ * Fluent interface.
+ * 
+ * @author Bruce Skingle
+ *
+ */
+public class EntityJsonParserBuilder
```

Comment:
That would break the code which calls it. I suppose I could then add a 

public static EntityJsonParserBuilder newInstance()

method to call the now private constructor, but for what? What's with the obsession about private constructors?